modules/import/csv_file.py

```diff
@@ -149,8 +149,8 @@ def __parse_file(self):
 
     def __register_options(self):
         # remove any old csv-file options
-        for option in self.options.keys():
-            if option.startswith('csv_'):
+        for option in list(self.options.keys()):
```

Comment:
Fixed issue with the python 2 to 3 conversion. This must iterate over a copy of the keys as it is modifying the dictionary within the loop. Otherwise, python throws an exception.

Reply:
Good catch, this is one of those 2to3 conversions that is unnecessary most of the time, but definitely a case where it needed to be done. Thanks!